manifests/init.pp

```diff
@@ -1,24 +1,49 @@
-# @summary A short summary of the purpose of this class
+# Manage your next generation Pulp server
 #
-# A description of what this class does
+# === Parameters:
```

Comment:
You are now converting it to rdoc rather than puppet-strings. That will provide the option to hide parameters if we expose it in the installer, but let's start with puppet-strings style documentation:

```
# @param cache_dir
#   Pulp cache directory
```

You can also leave out the defaults. Try running `bundle exec rake strings:generate:reference` and what `REFERENCE.md` contains. That will be the documentation we'll add on release (but keep out of git).

Reply:
Done. Thanks for the explanation.